Kinvey/KinveyTests/SyncStoreTests.swift

```diff
@@ -1342,4 +1342,23 @@ class SyncStoreTests: StoreTestCase {
         
     }
     
+    func testQueryWithPropertyNotMapped() {
+        let query = Query(format: "propertyNotMapped == %@", 10)
+        
+        weak var expectationFind = expectation(description: "Find")
+        
+        store.find(query) { persons, error in
```

Comment:
right, but since the problem was happening in the Realm layer, I first wrote a unit test that actually does reproduce the problem and then I re-run the same test again to make sure the unit test now passes without error.